src/Umbraco.Web/Search/UmbracoTreeSearcher.cs

```diff
@@ -85,6 +85,7 @@ public class UmbracoTreeSearcher
                     break;
                 case UmbracoEntityTypes.Media:
                     type = "media";
+                    fields = new[] { UmbracoExamineIndex.UmbracoFileFieldName };
```

Comment:
Is this now replacing all of the previous fields that it also must search on?

Reply:
Well, these are in addition to nodename but that does mean I've removed searching by id (I didn't know you could do that), so I'll put the id fields back!

Reply:
Interestingly you can't search by key, I'll see if I can add that also!

Reply:
Also now trying to figure out how to allow searching when there is a hyphen in the file name, e.g. "jan-copy.jpg" as it fails when the hyphen is reached....